howto/landing.md

```diff
@@ -14,5 +14,6 @@ The how-to guides in this section give directions on how to achieve your goals w
 | [How to implement streaming](https://discourse.ubuntu.com/t/implement-streaming/24332) | Instructions for using the Streaming Stack to implement streaming features in your application |
 | [How to manage cluster nodes](https://discourse.ubuntu.com/t/manage-cluster-nodes/24334) | Instructions for distributing the load of your Anbox Cloud installation over several machines in a cluster |
 | [How to troubleshoot Anbox Cloud](https://discourse.ubuntu.com/t/anbox-cloud-faq/17837)| Answers to common questions and issues |
+| [How to build an Anbox Cloud bug report](tbd)| Instructions for building and retrieving Anbox Cloud bug reports |
```

Comment:
Not needed if we move the page under the troubleshooting section.

Reply:
Fixed (rebased)